web/thesauruses/python/3/queues_stacks.json

```diff
@@ -277,13 +273,13 @@
       "queue_export_to_list": {
         "name": "Export a list of all queue items",
         "code": [
-          "mylist = queue"
+          "mylist = stack"
         ]
       },
```

Comment:
It looks like you're just copying stack over to mylist, but also this is for a queue. I'm a bit confused by this.